interfaces/builtin/appstream_metadata.go

```diff
@@ -112,5 +112,7 @@ func init() {
 		summary:              appstreamMetadataSummary,
 		implicitOnClassic:    true,
 		baseDeclarationSlots: appstreamMetadataBaseDeclarationSlots,
+
+		disruptiveForPlugOnRefresh: true,
```

Comment:
each of this probably should be accompanied by a small comment about some details of the use of the mount backend by the interface